addons/website/static/src/js/utils.js

```diff
@@ -5,6 +5,7 @@ var ajax = require('web.ajax');
 var core = require('web.core');
 
 var qweb = core.qweb;
+var _t = core._t;
```

Comment:
Avoid using var in new code. You can also use object destructuring if you would like:

```js
const {qweb, _t} = core;
```

to combine it with the qweb line